lib/filters.js

```diff
@@ -0,0 +1,66 @@
+var _ = require('lodash'),
+    moment = require('moment');
+
+/**
+ * filter items
+ * Add your filters to the fi obj as methods. See examples below.
+ * "fs" being short allowing tidier composition
+ */
+var fi = {};
+
+/**
+ * creates rearranges values and creates new date object
+ * @param  {String} d   A date string (must be) formatted (d)d/(m)m/yyy - in parens means optional
+ * @return {String}     a javascript date string
+ */
+fi.newDate = function date(d) {
+	var dateArr = d.split('/');
+	return dateArr.length === 3 ? new Date(dateArr[2], parseInt(dateArr[1]) - 1, dateArr[0]) : NaN;
+};
+
+/**
+ * returns a standard gov.uk date from a string using momentjs
+ * moment documentation: http://momentjs.com/docs/
+ * @method function
+ * @param  {string} d date e.g 09/12/1981 or 9-12-1981
+ * @param  {string} f moment.js format string (to override the default if needed)
+ * @return {string} date string as per the current gov.uk standard 09/12/1981 -> 09 December 1981
+ */
+fi.formatDate = function(d,f) {
+  return moment(fi.newDate(d)).locale('en-gb').format(f ? f : 'LL');
+};
+
+/**
+ * logs an object in the template to the console on the client.
+ * @param  {Any} a any type
+ * @return {String}   a script tag with a console.log call.
+ * @example {{ "hello world" | log }}
+ * @example {{ "hello world" | log | safe }}  [for environments with autoescaping turned on]
```

Comment:
Have you turned auto escaping off? Do you think that might be a better default for the kit? Otherwise log will always have to be followed by safe, is it possible to just call log and incorporate the safe behaviour?